src/tekster/finn-tekst.tsx

```diff
@@ -18,11 +18,11 @@ export function finnTekst(id: string, language: Locale, payload?: string): strin
         case Locale.UKRAINSK:
         case Locale.RUSSISK:
             id += '-en';
-            ledetekst = ledetekster[id];
+            ledetekst = ledetekster[id as LangKey];
```

Comment:
Er en feil i ledetekstene som TS klikker på, se id for en/se-versjonene av `varslet-epost-og-sms`.

Hvis den fikses kan det gjøres det sånn for å slippe castingen 🙂 
```suggestion
            ledetekst = ledetekster[`${id}-en`];
```